frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Table.js

```
@@ -159,156 +155,157 @@ const Table = ({
-            </tbody>
-          </table>
+                ))}
+              </thead>
+              <tbody {...getTableBodyProps()}>
+                {page.map((row, i) => {
+                  prepareRow(row);
+                  return (
+                    <tr {...row.getRowProps()} onClick={() => onClickRow(row)} className={rowClassName}>
+                      {showAutoSerialNo && <td>{i + 1}</td>}
+                      {row.cells.map((cell) => (
+                        <td {...cell.getCellProps([getCellProps(cell)])}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)